docs/preact-testing-library/api.md

```diff
@@ -0,0 +1,273 @@
+---
+id: api
+title: API
+---
+
+`preact-testing-library` re-exports everything from `dom-testing-library` as
+well as these methods:
+
+- [`render`](#act)
+- [`cleanup`](#cleanup)
+- [`act`](#act)
```

Comment:
- I see `debounceRenderingOff` and `flushPromises` methods that are not mentioned here
- `act` is not in the Preact implementation